### PR TITLE
Ramya unit test: Userprofile/TeamWeeklySummaries

### PR DESCRIPTION
**Pull request by scoopstroop92**
# Description
Unit tests for src/components/UserProfile/TeamWeeklySummaries

## Main changes explained:
-  New file
      - **TeamWeeklySummaries.test.js** : unit test cases 
-  Existing file
      - **TeamWeeklySummaries**  : Added  test id to the objects for unit testing 
####  Test cases 
image

## How to test:
1. check into current branch
2. do npm test TeamWeeklySummaries.test.jsx
3. check if all the test cases pass

## Screenshots or videos of changes:
image

## Note : 
Kindly take a look at the test cases and  suggest if more validations can be added

### CODE REVIEW

**Comment by KurtisIvey** [comment]
a test file in the proper location. Brings a tear of happiness to my eye 🥲

**Comment by hdani18** [comment]
There is a typo in the above mentioned command to run the file:
Correct cmd is : `npm test TeamWeeklySummaries.test.js`

**Comment by belagaam** [comment]
i tested this PR and I got an error while running the npm test command.

Screenshot 2024-01-06 at 5 40 37 PM

**Review by hdani18 — DISMISSED** [review]
Screen Shot 2024-01-06 at 5 39 20 PM
All test cases passed LGTM!

**Review by bienzguoa — DISMISSED** [review]
All test cases passed! It looks very good, with all rendering thoroughly and correctly.

Screenshot 2024-01-06 at 4 37 45 PM

**Comment by hdani18** [comment]
> i tested this PR. All test cases passed
> 
> PR 1785 - VS Working

**Review by AaronPersaud — DISMISSED** [review]
Hi Ramya, I've ran the tests and everything looks good. Nice work!

image

**Comment by hdani18** [comment]
I re-ran this locally and all the test cases have passed locally, but fails in the workflow. @ramyaram2092  please have a look.

**Failed on GitHub workflow:**
Screen Shot 2024-01-06 at 10 00 02 PM

**Output on Local:**
Screen Shot 2024-01-06 at 9 54 45 PM

**Comment by ramyaram2092** [comment]
> There is a typo in the above mentioned command to run the file: Correct cmd is : `npm test TeamWeeklySummaries.test.js`

Updated. Thank you !

**Comment by ramyaram2092** [comment]
> I re-ran this locally and all the test cases have passed locally, but fails in the workflow. @ramyaram2092 please have a look.
> 
> **Failed on GitHub workflow:** Screen Shot 2024-01-06 at 10 00 02 PM
> 
> **Output on Local:** Screen Shot 2024-01-06 at 9 54 45 PM

Checking that now. Marking the PR "do not review" on the same note

File: src/components/UserProfile/TeamWeeklySummaries/TeamWeeklySummaries.jsx
Comment (hdani18) [inline src/components/UserProfile/TeamWeeklySummaries/TeamWeeklySummaries.jsx:1]: Hi @ramyaram2092. There is still one test case that is failing. Please have a look. Thanks!

Screen Shot 2024-01-09 at 6 48 11 PM

**Review by hdani18 — COMMENTED** [review]
Hi @ramyaram2092. There is still one test case that is failing. Please have a look. Thanks!
Screen Shot 2024-01-09 at 6 48 11 PM

**Comment by DragonSlayer2401** [comment]
One of the test cases failed please watch the overview video below for more details: 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102876103/7a85ed36-0761-4aff-8e3e-3a60f75e67e6

**Comment by ramyaram2092** [comment]
Closing this PR and creating a new one . Kindly test [1820](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1820)

**Pull request closed without merging**